contracts/evm/scripts/sophon-push.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -eE
+
+# Get the auth token from an arg
+FEEDS=$1
+if [ -z "$FEEDS" ]; then
+  echo "Feeds are required"
+  exit 1
+fi
+
+AUTH_TOKEN=$2
+if [ -z "$AUTH_TOKEN" ]; then
+  echo "Auth token is required"
+  exit 1
+fi
+
+SLEEP_INTERVAL=${3:-60}
+
+# Loop indefinitely
+while true; do
+  # Capture the start time
+  start_time=$(date +%s)
+
+  # Run the command
+  npx hardhat --network sophonMainnet interact updateTemporalNumericValuesV1 $FEEDS https://rest.jp.stork-oracle.network $AUTH_TOKEN --paymaster-address 0x98546B226dbbA8230cf620635a1e4ab01F6A99B2
```

Comment:
ah is this meant to stand in for the EVM pusher? or is it just for testing?

Reply:
This is just for testing sophon since they have this paymaster thing. Ideally evm would use the go pusher in the repo